resources/conf_file.rb

```diff
@@ -1,35 +1,15 @@
-
 #
 # Cookbook:: auditd
 # Resource:: auditd_conf_file
-#
```

Comment:
We do actually want this. I think the original comment saying to remove it in confusion.